CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,168 @@
+
+The GNOME Code of Conduct is also included at the end of this file.
+
+
+GNOME Code of Conduct
+
+Thank you for being a part of the GNOME project. We value your participation and want everyone to have an enjoyable and fulfilling experience. Accordingly, all participants are expected to follow this Code of Conduct, and to show respect, understanding, and consideration to one another. Thank you for helping make this a welcoming, friendly community for everyone.
+
+Scope
+
+This Code of Conduct applies to all GNOME community spaces, including, but not limited to:
+
+    Issue tracking systems - bugzilla.gnome.org
+    Documentation and tutorials - developer.gnome.org
+    Code repositories - git.gnome.org and gitlab.gnome.org
+    Mailing lists - mail.gnome.org
+    Wikis - wiki.gnome.org
+    Chat and forums - irc.gnome.org, discourse.gnome.org, GNOME Telegram channels, and GNOME groups and channels on Matrix.org (including bridges to GNOME IRC channels)
+    Community spaces hosted on gnome.org infrastructure
+    Any other channels or groups which exist in order to discuss GNOME project activities
+    All event venues and associated spaces, including conferences, hackfests, release parties, workshops and other small events
+    All areas related to event venues: vendor exhibit halls, staff and meal areas, connecting infrastructure like walkways, hallways, elevators, and stairs
+    Sponsor events, either on-site or off-site
+    Private events off-site that involve one or more attendees
+    Private conversations taking place in official conference hotels 
+
+Communication channels and private conversations that are normally out of scope may be considered in scope if a GNOME participant is being stalked or harassed. Social media conversations may be considered in-scope if the incident occurred under a GNOME event hashtag, or when an official GNOME account on social media is tagged, or within any other discussion about GNOME. The GNOME Foundation reserves the right to take actions against behaviors that happen in any context, if they are deemed to be relevant to the GNOME project and its participants.
+
+All participants in GNOME community spaces are subject to the Code of Conduct. This includes GNOME Foundation board members, corporate sponsors, and paid employees. This also includes volunteers, maintainers, leaders, contributors, contribution reviewers, issue reporters, GNOME users, and anyone participating in discussion in GNOME community spaces. For in-person events, this also includes all attendees, exhibitors, vendors, speakers, panelists, organizers, staff, and volunteers.
+
+Reporting an Incident
+
+If you believe that someone is violating the Code of Conduct, or have any other concerns, please contact the Code of Conduct committee.
```

Comment:
change to "please contact us via [email](mailto:heli@guy.4577). Alternatively, use [Matrix](link.tld)" or similar?